### PR TITLE
doc: man/8/rados.rst: make the description clearer

### PR DESCRIPTION
**Pull request by drunkard**
Signed-off-by: Drunkard Zhang <gongfan193@gmail.com>

### CODE REVIEW

**Comment by neha-ojha** [comment]
jenkins render docs

File: doc/man/8/rados.rst [hunk 1]
```diff
@@ -202,10 +202,10 @@ Pool specific commands
   Set the value of the object map header of object name.
 
 :command:`export` *filename*
-  Serialize pool contents to a file or standard output.\n"
+  Serialize pool contents to a file or standard output.
 
 :command:`import` [--dry-run] [--no-overwrite] < filename | - >
-  Load pool contents from a file or standard input
+  Load contents from a file or standard input into a pool.
```
Comment (tchaikov) [inline doc/man/8/rados.rst]: but i think "pool contents" here is consistent with the notation used by the usage of "export" subcommand, shall we change it to "contents"? if we really want to do so, probably we could be consistent on this?

Comment (drunkard) [reply doc/man/8/rados.rst]: I was thinking, before loaded to pool, it's not "pool contents" yet, so I made the change.

Comment (drunkard) [reply doc/man/8/rados.rst]: Accuracy is important than consistent, right?

**Pull request closed without merging**